app/connectors/parsers/PensionSchemeParser.scala

```diff
@@ -59,4 +61,39 @@ object PensionSchemeParser {
           }
       }
   }
+
+  implicit object GetAuthorisingPsaIdHttpReads
+      extends HttpReads[AuthorisingPsaIdType]
+      with Logging
+      with DownstreamLogging {
+
+    private val authorisingPsaIdFromApiReads: Reads[PsaId] =
```

Comment:
This won't read anything from the returned Json from pensions-scheme https://github.com/hmrc/pensions-scheme-stubs/blob/main/conf/resources/data/getPspSchemeDetails/21000005/24000015IN.json#L91

This is returned from pensions-scheme here - https://github.com/hmrc/pensions-scheme/blob/main/app/controllers/SchemeDetailsController.scala#L130